templates/al2/provisioners/install-worker.sh

```diff
@@ -303,7 +303,7 @@ for binary in "${BINARIES[@]}"; do
   fi
   sudo sha256sum -c $binary.sha256
   sudo chmod +x $binary
-  sudo mv $binary /usr/bin/
+  sudo mv -Z $binary /usr/bin/
```

Comment:
if we're conforming to this, we should probably add some kind of validation as well.

I'm also wondering if it would be better to do something more general to apply expected selinux config, like:

```bash
# enforce correct SELinux perms in /usr/bin
semanage fcontext -a -t bin_t "/usr/bin(/.*)?"
restorecon -R -v /usr/bin
```

Reply:
nit: use the long option name in scripts for clarity
```suggestion
  sudo mv --context $binary /usr/bin/
```

Reply:
> I'm also wondering if it would be better to do something more general to apply expected selinux configs at the end of the build

The best place would be in the template.json at the end of `provisioners`
or as part of `post-processors`. Something like:

```
{
      "type": "shell",
      "inline": [
         "semanage fcontext -a -t bin_t \"/usr/bin(/.*)?\"",
         "restorecon -R -v /usr/bin"      
       ]
    }
```
What do you think?